code-challenges.test.js

```diff
@@ -25,14 +31,65 @@
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+describe('When fibLength is called', () => {
+  it('returns the the input as the length of the array', () => {
+    //Arrange 
+    const numArgument = 6
+    
+    //Act
+    const actualResult = fibLength(numArgument)
+    
+    //Assert
+    expect(actualResult).toEqual([1, 1, 2, 3, 5, 8])
+  })
+
+  it('returns the the input as the length of the array', () => {
+    //Arrange 
+    const numArgument = 10
+    
+    //Act
+    const actualResult = fibLength(numArgument)
+    
+    //Assert
+    expect(actualResult).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
 
+// b) Create the function that makes the test pass.
 
+// const fibLength = (inputForLength) => {
+//   let lengthOfArray = new Array(inputForLength).fill(null)
+//   return lengthOfArray.map(value, index => {
+//     let 
+//   })
+// } 
 
-// b) Create the function that makes the test pass.
 
+// const fibLength = (inputForLength) => {
+//   let storageArray = []
+//   let a = 0
+//   let b = 1
+//   for(let i = 0; i <= inputForLength + 1; i++){
+//     b = a + b
+//     console.log(a,b)
+//     storageArray.push(b)
+//     a = storageArray[i]
+//   }
+//   return storageArray
+// }
+
+const fibLength = (inputForLength) => {
```

Comment:
nice and concise. What if the function is called with 0 or 1? Seems this function would still return `[1,1]` which wouldn't have the desired length